h/static/styles/simple-search.scss

```diff
@@ -23,14 +23,13 @@
 }
 
 :not(:focus) ~ .simple-search-icon {
-  color: $gray-lighter;
+  color: $gray-light;
 }
 
 input.simple-search-input {
   float: left;
   outline: none;
   color: $text-color;
-  width: 100%;
```

Comment:
Are we sure this causes no problems? I thought it was necessary or we'd end up with a short field.